functests/cincinnati_creation_test.go

```diff
@@ -5,34 +5,15 @@ import (
 	"crypto/tls"
 	"fmt"
 	"net/http"
-	"os"
 	"testing"
 
-	routev1 "github.com/openshift/api/route/v1"
 	cincinnativ1beta1 "github.com/openshift/cincinnati-operator/pkg/apis/cincinnati/v1beta1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/util/wait"
-	"k8s.io/client-go/kubernetes/scheme"
-	"k8s.io/klog"
-	client "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-// TestMain will perform test setup, exec each test, and perform test tear down.
-func TestMain(m *testing.M) {
-	setUp()
-	retCode := m.Run()
-	os.Exit(retCode)
-}
-
-// setUp performs test setup.
-func setUp() {
-	if err := routev1.AddToScheme(scheme.Scheme); err != nil {
-		klog.Fatalf("Failed adding route to scheme, %v", err)
-	}
-}
```

Comment:
Not sure why we do not need this?

Reply:
Because we are no longer calling the route API, now that the Cincy object is abstracting the route implementation.